test/both/transport/connection-test.js

```diff
@@ -320,7 +320,7 @@ describe('Transport/Connection', function() {
 
     it('should handle SETTINGS', function(done) {
       client._spdyState.framer.settingsFrame({
-        max_frame_size: 100,
+        max_frame_size: 16384,
```

Comment:
I don't really agree with this, but I will fix it myself. Don't want to waste your time on it! Thank you.